common/persistence/pinotVisibilityTripleManager.go

```diff
@@ -46,8 +46,9 @@ type (
 )
 
 const (
-	Primary   = "Primary"
-	Secondary = "Secondary"
+	Primary    = "Primary"
+	Secondary  = "Secondary"
+	ContextKey = types.ContextKey("visibility-override")
```

Comment:
Mind if I ask what your thinking was with creating a type for this? This might be a pattern we're doing elsewhere, I may have just missed it